src/mlpack/bindings/python/mlpack/matrix_utils.py

```diff
@@ -160,7 +160,10 @@ def to_matrix_with_info(x, dtype, copy=False):
       dims = len(x)
 
     d = np.zeros([dims])
-    out = np.array(x, dtype=dtype, copy=copy) # Try to avoid copy...
+    if copy:
+      out = np.asarray(x, dtype=dtype, copy=True)
+    else:
+      out = np.asarray(x, dtype=dtype, copy=None)
```

Comment:
Don't think `asarray` had the `copy` argument in NumPy 1. It isn't listed [in the NumPy 1 docs]( https://numpy.org/doc/1.26/reference/generated/numpy.asarray.html ). Also [NumPy 2 docs]( https://numpy.org/doc/2.0/reference/generated/numpy.asarray.html ) note `copy` is a new argument. Testing local with NumPy 1 am seeing an error. Am I missing something? Or are these meant to be `array` instead?

Reply:
Ahh, thank you!  I did mean to write `array()` there, that was the documentation I checked to come up with this fix.  Just pushed a fix now.

Reply:
All good 😊 Thanks Ryan 🙏